Makefile

```diff
@@ -82,16 +82,27 @@ endif
 checkout:
 	./scripts/checkout.sh
 
+#This could currently break if not using COSMOS in the config.
 ci-launch:
 	@export SYSTEM_TEST_FILE_PATH=$(SYSTEM_TEST_FILE_PATH) && \
 	./scripts/ci_launch.sh && \
+	./scripts/system_tests.sh && \
 	./scripts/stop.sh
 
+#This could currently break if not using COSMOS in the config.
 system-tests:
 	@export SYSTEM_TEST_FILE_PATH=../..$(SYSTEM_TEST_FILE_PATH) && \
 	./cfg/build/launch.sh && \
 	./scripts/system_tests.sh
 
+#Be sure that your nos3-mission.xml has been set to YAMCS
+yamcs-operator:
+	@export SYSTEM_TEST_FILE_PATH=$(SYSTEM_TEST_FILE_PATH) && \
+	./scripts/ci_launch.sh --use-yamcs
+
+cosmos-operator:
+	echo "TODO:  This is not yet functional"
```

Comment:
Can we get this one in here too to keep things simple?

Reply:
I thought I had pushed this up.  Sure thing.